### DISCUSSION TITLE
Welcome to orb9k_circuitpython Discussions!

### DISCUSSION THREAD

**Discussion by vputz** [thingotron/orb9k_circuitpython#1] [General]
## 👋 Welcome!
 Welcome SpaceOrb fans!  In the past, I've used my own hosted wiki, but now that all the software is handily open-source, it seems best to let discussions and wiki reside on Github.

If you have issues with the existing Orbotron software, use the issue tracker--if you have discussions, good configs, anything else, let's use this!

**Comment by twinankhs** [comment]
Hey - Probably the only one here - I was on the development team at Spacetec, but alas only a QA person for the wonderful SpaceOrb controller. I was trying to get it to work with some games and noticed steam support is not very good. Good news is if we can get hold of Alex Vlanchos and see if he can get it working - he worked at Spacetec with me and Jack Follansbee, Glen Peterson, Volkan Can and so many others. I knew the whole team pretty well as I had to break all their stuff. I have so many more stories to tell about Spacetec!

**Reply by gamer85** [reply]
Great stuff.  Any details on the development and QA period for the SpaceOrb would be interesting to hear.
